app/API/V1/Http/Controllers/ReparationRequestMaterialController.php

```diff
@@ -37,14 +39,37 @@ public function index(): ReparationRequestMaterialCollection
         return new ReparationRequestMaterialCollection($reparationRequestMaterial);
     }
 
-    public function store(ReparationRequestMaterialStoreRequest $reparationRequestMaterialStoreRequest): ReparationRequestMaterialResource
+    public function show(ReparationRequestMaterial $reparationRequestMaterial): ReparationRequestMaterialResource
     {
-        $reparationRequest = ReparationRequest::find((int) $reparationRequestMaterialStoreRequest->get('reparation_request_id'));
+        return new ReparationRequestMaterialResource($reparationRequestMaterial);
+    }
+
+    public function store(StoreReparationRequestMaterialRequest $request): ReparationRequestMaterialResource
+    {
+        $reparationRequest = ReparationRequest::find((int) $request->get('reparation_request_id'));
+
+        $reparationRequestMaterial = ReparationRequestMaterial::make($request->except(['reparation_request_id']));
```

Comment:
Do you know about the `$request->safe()` method? It will only return a collection of validated data. It's safer to use  here. See https://laravel.com/docs/9.x/validation#working-with-validated-input